### PR TITLE
Update App Button Component Style, Implement Lab Class

### LINKED ISSUE #153: Implement Lab Class
Please implement the Lab class. Comments on this file describe what makes a lab unique.

https://github.com/CarletonComputerScienceSociety/discretemath.ca/blob/master/app/src/apps/Examinations/lib/examinations/Lab.js

### PR DESCRIPTION
**Pull request by bininglong**
Fixes #153

### CODE REVIEW

File: app/src/pages/index.astro [hunk 1]
```diff
@@ -12,8 +12,8 @@ import { Button } from "../components";
     mathematics courses.
   </div>
   <div>
-    <Button label="comp 1805" />
-    <Button label="comp 2804" />
+    <Button label="COMP 1805" />
+    <Button label="COMP 2804" />
```
Comment (MathyouMB) [inline app/src/pages/index.astro:16]: Instead of buttons caps in the label, we can use a `text-transform: capitalize;`

**Review by MathyouMB — CHANGES_REQUESTED** [review]
What is the added `discretemath.ca` submodule for? I'm confused at what this is as it makes it sound like it's making the project a submodule to itself.

**Comment by AngelOnFira** [comment]
I imagine the submodule addition was just a mistake clone, I can make a fix for that if you like @bininglong :)

**Comment by bininglong** [comment]
> I imagine the submodule addition was just a mistake clone, I can make a fix for that if you like @bininglong :)

Sure, thank you!

File: app/src/pages/index.astro (hunk 1 above)
Comment (bininglong) [reply app/src/pages/index.astro:16]: Yes, that's better, I will fix it!

**Review by MathyouMB — APPROVED** [review]

**Pull request merged**